common/src/main/java/org/commonjava/maven/ext/common/util/GAVUtils.java

```diff
@@ -19,17 +19,18 @@
 import org.commonjava.maven.atlas.ident.ref.ProjectVersionRef;
 import org.jboss.da.model.rest.GAV;
 
-import java.util.ArrayList;
+import java.util.HashSet;
 import java.util.List;
+import java.util.Set;
 
 /**
  * Commonly used manipulations from project profiles.
  */
 @UtilityClass
 public final class GAVUtils
 {
-    public static List<GAV> generateGAVs( List<ProjectVersionRef> dep) {
-        ArrayList<GAV> result = new ArrayList<>();
+    public static Set<GAV> generateGAVs( List<ProjectVersionRef> dep) {
+        Set<GAV> result = new HashSet<>();
```

Comment:
Maybe order does not matter here, but the original `List` is ordered, so wondered if we'd want to maintain the order and use `LinkedHashSet` instead.

Reply:
I don't believe order matters , I think it's just too pass a collection (of some form) through to DA

Reply:
Its for https://github.com/project-ncl/dependency-analysis/blob/master/reports-model/src/main/java/org/jboss/da/lookup/model/MavenLookupRequest.java#L20